core/modules/search/tests/search.test

```diff
@@ -1380,15 +1380,15 @@ class SearchConfigSettingsForm extends BackdropWebTestCase {
 
     // Test that the settings form displays the correct count of items left to index.
     $this->backdropGet('admin/config/search/settings');
-    $this->assertText(t('There are @count items left to index.', array('@count' => 0)));
+    $this->assertText(t('@count nodes left to index or excluded from indexing.', array('@count' => 0)));
```

Comment:
This "or excluded" part seems like it is not very helpful. If I exclude a content type that has 100 items in it, will this message say that there are always 100 nodes left to index or excluded? As a site-admin, I want to know which is which. 100% indexing of enabled content types even when others are excluded is important. So I'd recommend this messaging be updated to say something like `@count content items of type @type are left to index. @excluded_count are excluded.` We also use the word "content" instead of "node" in UIs.